doc/source/user_guide/advanced.rst

```diff
@@ -869,12 +869,12 @@ implementing an ordered, sliceable set.
 Float64Index
 ~~~~~~~~~~~~
 
-.. note::
-
-    In pandas 2.0, :class:`NumericIndex` will become the default index type for numeric types
+.. deprecated:: 1.4.0
```

Comment:
does this need a blank line after?

Reply:
No. It's used in in used e.g. in user_guide/io.rst, where it works fine.